core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerCreator.java

```diff
@@ -18,8 +19,8 @@
         this.trackSelector = trackSelector;
     }
 
-    public SimpleExoPlayer create() {
-        DefaultRenderersFactory renderersFactory = new DefaultRenderersFactory(context);
+    public SimpleExoPlayer create(DrmSessionCreator drmSessionCreator) {
```

Comment:
It should be fine to move as @juankysoriano says. We are creating it in the `ExoPlayerTwoImplFactory` and it is only passed through the constructor, stored and use only for the internal `loadVideo`. We don't give an option to the client to pass through a different `DrmSessionCreator` we need to create from the `ExoPlayerTwoImplFactory`.